### ISSUE TITLE
Relocating EasyStack files to a separate repository

### ISSUE THREAD

**Issue by ocaisa** [EESSI/software-layer#300]
Right now, to add support for new software a PR needs to be made to this repo. This repo though is quite powerful as it also contains all of the machinery that we use to actually do the installations. It is probably better to move the easystack files to a separate repository, so that we can have a greater separation of concerns, and enforce tighter policies on this repository.

Given that we have made a choice to do end user support via GitLab. Moving the easystack files to a GitLab repository may also make sense, since we can keep all the end-user facing aspects of EESSI in one place (and reduce the cross-referencing between different platforms).

**Comment by boegel** [comment]
I agree it makes sense to split off the easystack files to a separate repo.

I can see the value in moving to GitLab, but to some extent that may also have an impact on contributors, since I'm pretty sure people are more likely to be familiar with GitHub than GitLab.
For the support portal (issues, or emails) that doesn't matter too much, but for actual "code" contributions, it may...

And of course, before that's feasible, the bot needs to be able to talk to GitLab (cfr. https://github.com/EESSI/eessi-bot-software-layer/issues/194)
